docs/book/migration/to-v3-0.md

```diff
@@ -0,0 +1,84 @@
+# Upgrading to 3.0
+
+## Module autoloading
```

Comment:
The migration guide is very good, but only here the autoloading is included. No code example for autoloading in the other documents. Only a note in the introduction.
I think a code example (`composer.json`) in the module class chapter will help a lot.